controllers/operands/kubevirtConsolePlugin_test.go

```diff
@@ -292,6 +292,59 @@ var _ = Describe("Kubevirt Console Plugin", func() {
 			Expect(hco.Status.RelatedObjects).To(Not(ContainElement(*objectRefOutdated)))
 			Expect(hco.Status.RelatedObjects).To(ContainElement(*objectRefFound))
 		})
+
+		Context("Node Placement", func() {
```

Comment:
@tiraboschi one spec I noticed in `ssp_test.go` was
```go
It("should overwrite node placement if directly set on SSP CR", func()
```
It got me wondering if similar behaviour should exist for kubevirt-console-plugin. At the moment, manually editing the kubevirt-console-plugin deployment to change its node placement configuration doesn't trigger anything from HCO side. Should HCO reconcile it if a user manually changes this setting?